dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -568,6 +569,9 @@ private void updateInfo(Workflow oldWorkflow, Workflow newWorkflow) {
             oldWorkflow.setTopicSelection(newWorkflow.getTopicSelection());
         }
 
+        // Update DOI selection
+        oldWorkflow.setDoiSelection(newWorkflow.getDoiSelection());
```

Comment:
Should we check the selection against the existing set of DOIs, and do something like reject if it is inconsistent?

Reply:
Updated it so it ignores the DOI selection if there are no existing DOIs of that type